cli/src/infragpt/container.py

```
@@ -0,0 +1,290 @@
+            except KeyboardInterrupt:
+                self.cancelled = True
+                console.print("\n[yellow]Command cancelled by user[/yellow]")
+                try:
+                    self.client.api.exec_start(
+                        self.client.api.exec_create(
+                            container=self.container.id,
+                            cmd=["/bin/sh", "-c", "pkill -P 1"],
+                        )
+                    )
+                except Exception:
+                    pass
```

Ruff (0.14.8)

`try`-`except`-`pass` detected, consider logging the exception

(S110)